policyengine_us/variables/gov/usda/snap/income/deductions/snap_child_support_deduction.py

```diff
@@ -16,7 +16,13 @@ class snap_child_support_deduction(Variable):
     # calculation
     def formula(spm_unit, period, parameters):
         child_support = add(spm_unit, period, ["child_support_expense"])
+        # Exclude ineligible members' share per SNAP proration rules (result always >= 0 since prorate_fraction < spm_unit_size)
+        prorate_fraction = spm_unit("snap_prorate_fraction", period.this_year)
```

Comment:
I believe prorating should be done on a person level -- We want to check what the child support amount of the ineligible person is and then give a percentage of that to the unit 

Can we add a test along the lines of:
person 1 (ineligible): child support: $1000
person 2 (eligible): child support: $300 

total child support for deduction: $800

Reply:
Dont think the deduction itself should be prorated